src/services/CashMachine.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios')
+const qs = require('querystring')
+const ora = require('ora')
+
+const resultOutput = require('../utils/result')
+const errorOutput = require('../utils/error')
+const unExpectedError = require('../utils/unExpectedError')
+
+const apiConfig = require('../config/api')
+
+const setUrl = (url) => {
+  return url
+}
+
+const setRequestBody = (userInput) => {
+  return qs.stringify({ amount: userInput })
+}
+
+const setHeader = (contentType) => {
+  return { 'headers': setContentType(contentType) }
+}
+
+const setContentType = (contentType) => {
+  return { 'Content-Type': contentType }
+}
+
+module.exports = (userInput) => {
```

Comment:
this is ok but I think it could be more readable it will be service/class but the functional way is also great.